static/openapi/prerendering-spec.json

```diff
@@ -2059,999 +1891,1491 @@
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "description": "Bad request.",
+                  "properties": {
+                    "status": {
+                      "type": "string",
+                      "description": "Error status."
+                    },
+                    "message": {
+                      "type": "string",
+                      "description": "Error message."
+                    }
+                  },
+                  "required": ["status", "message"],
+                  "example": {
+                    "status": "ERROR",
+                    "message": "Header x-photoniq-customerid is required"
+                  }
                 }
               }
             }
           },
           "500": {
-            "description": "Internal server error",
+            "description": "Internal server error.",
             "content": {
               "application/json": {
                 "schema": {
                   "type": "object",
-                  "description": "Internal server error",
+                  "description": "Internal server error.",
                   "properties": {
                     "status": {
                       "type": "string",
-                      "description": "Error status"
+                      "description": "Error status."
                     },
                     "message": {
                       "type": "string",
-                      "description": "Error message"
+                      "description": "Error message."
                     }
                   },
-                  "required": ["status", "message"]
+                  "required": ["status", "message"],
+                  "example": {
+                    "status": "ERROR",
+                    "message": "Internal server error"
+                  }
                 }
               }
             }
           }
         }
       }
     },
-    "/api/prerender/v1/origins": {
+    "/api/prerender/v1/metrics/percentiles/responsetime": {
       "get": {
-        "summary": "Get origins",
-        "tags": ["Origins"],
-        "description": "Returns all origins.",
+        "summary": "Get percentile metrics ",
+        "tags": ["Metrics"],
+        "description": "Returns percentile metrics for response times within a specific time range, optionally filtered by URL pattern and response code. Required query parameters include from and to, denoting the epoch time range in seconds. An optional urlRegex parameter allows filtering by URL patterns, while responseCode filters the metrics by HTTP response code. The endpoint returns the 90th, 95th, and 99th percentiles of response times. A customerId header is required for authentication.",
```

Comment:
This description is different from all other descriptions.  Here the description describes all the input parameters and what it returns, but in none of the other APIs this is done.  Why is this different here?

The required fields are flagged below ("required").  So,  probably don't need to explain that.

The swagger defines what is returned in the response section.

Why does this mention the customerId header for authentication?  Do the other APIs not need this?

Reply:
Correct. Please refer to [Allowlist-Based Header Values](https://www.macrometa.com/docs/photoniq/prerendering/prerendering-headers/allowlist-value-headers) for more information.

Reply:
Yeah, I wonder if we should be too detailed in the endpoint description because lots of information here will be in the description of `attributes` / `headers` / `query params` / `path params`.
I will take this topic to the developers.